wallet-ext: update max height

pchrysochoidis opened the pull request:
## Description 

* now wallet can expand to 680px if there is available room


https://github.com/MystenLabs/sui/assets/10210143/b9e7ed20-e618-4fcf-b181-da91e150bdc9


closes APPS-1637

## Test Plan 

👀 

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### Release notes

mystie711 reviewed (approved):
Looking good.

If there was a way to keep the graphic center aligned vertically in the space between top/bottom that would be preferable.

pchrysochoidis commented:
> Looking good.
> 
> If there was a way to keep the graphic center aligned vertically in the space between top/bottom that would be preferable.

@mystie711  I updated it, added new video in the description
